modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/admin/KeyManagersTestCase.java

```diff
@@ -1104,8 +1146,93 @@ public void testAddKeyManagerWithExistingKeyManagerName() throws ApiException {
         }
     }
 
+    @Test(groups = {"wso2.am"}, description = "Test key manager permissions with WSO2IS with permissions"
+            ,dependsOnMethods = "testDeleteKeyManagerWithAuth0")
+    public void testKeyManagerPermissions() throws Exception {
+
+        userManagementClient.addUser(USER_TEST, USER_TEST_PASSWORD, ROLE_LIST, USER_TEST);
```

Comment:
Shall we move role user creation to the setup or pre-test method. Because, if the test case was skipped, it tries to remove a nonexisting role or user